net.sourceforge.atanks.json

```diff
@@ -0,0 +1,111 @@
+{
+    "app-id": "net.sourceforge.atanks",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--persist=.atanks",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "rename-desktop-file": "atanks.desktop",
+    "rename-icon": "atanks",
+    "modules": [
+        "shared-modules/glu/glu-9.0.0.json",
+        {
+            "name": "alsa-lib",
```

Comment:
The version in the SDK is compiled without the rawmidi API, which is required by this app. That's why I rebuild a new version here.